blazor/datagrid/tool-bar.md

```diff
@@ -577,4 +577,72 @@ You can able to customize the toolbar styles by using the toolbar class (`.e-too
 The following screenshots represent a datagrid by customizing toolbar styles.
 ![Customizing ToolBar Styles in Blazor DataGrid](./images/blazor-datagrid-toolbar-style-customization.png)
 
-> You can refer to our [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid) feature tour page for its groundbreaking feature representations. You can also explore our [Blazor DataGrid example](https://blazor.syncfusion.com/demos/datagrid/overview?theme=bootstrap4) to understand how to present and manipulate data.
\ No newline at end of file
+## Customizing the toolbar items tooltip text
+
+You can able to customize the toolbar items tooltip text by using the [ItemModel's](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.ItemModel.html#Syncfusion_Blazor_Navigations_ItemModel__ctor) `TooltipText` property.
```

Comment:
You can customize the toolbar items tooltip text by adding toolbar items externally by setting  [ItemModel.TooltipText](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.ItemModel.html#Syncfusion_Blazor_Navigations_ItemModel_TooltipText) property.